### PR TITLE
sdl2: macos is non unix

### PR DESCRIPTION
**Pull request by Jan200101**
tries to fix the build issue found in #721

### CODE REVIEW

**Comment by Jan200101** [comment]
@neheb can you look over to verify I didn't mess anything up and that it works as intended now?

File: ci_config.json [hunk 1]
```diff
@@ -414,6 +414,11 @@
       "libjack-dev"
     ]
   },
+  "sdl2": {
```
Comment (neheb) [inline ci_config.json:417]: I’d get rid of this honestly.

Comment (Jan200101) [reply ci_config.json:417]: yeah, Its just there to verify the CI works, I'll remove it as soon as we know this is solid.

Comment (neheb) [reply ci_config.json:417]: Seems fine now.

**Comment by Jan200101** [comment]
For documentation, this is the passing MacOS CI build for this before I removed the dbus installation
https://github.com/mesonbuild/wrapdb/actions/runs/3340282381/jobs/5530207162

**Pull request merged**